### ISSUE TITLE
How can I turn off the extra output information (Only keep PoS information) #SyntaxNet#

### ISSUE THREAD

**Issue by ColinUNCG** [tensorflow/models#230]
`echo 'Bob brought the pizza to Alice.' | syntaxnet/demo.sh`

**How can I remove the extra output information** 
> INFO:tensorflow:Processed 1 documents
> INFO:tensorflow:Total processed documents: 1
> INFO:tensorflow:num correct tokens: 0
> INFO:tensorflow:total tokens: 7
> INFO:tensorflow:Seconds elapsed in evaluation: 0.14, eval metric: 0.00%
> INFO:tensorflow:Processed 1 documents
> INFO:tensorflow:Total processed documents: 1
> INFO:tensorflow:num correct tokens: 1
> INFO:tensorflow:total tokens: 6
> INFO:tensorflow:Seconds elapsed in evaluation: 0.59, eval metric: 16.67%
> INFO:tensorflow:Read 1 documents
> Input: Bob brought the pizza to Alice .

**Only keep the PoS information** 
> brought VBD ROOT
>  +-- Bob NNP nsubj
>  +-- pizza NN dobj
>  |   +-- the DT det
>  +-- to IN prep
>  |   +-- Alice NNP pobj
>  +-- . . punct

**I modified the demo.sh file. I removed some extra output information, how can I Only keep the PoS information? Thanks**

**Comment by smaransmarty** [comment]
It would help if you show your demo.sh file and be specific on what you mean by only PoS information.

Syntaxnet has two parts - PoS Tagger and a Dependency Parser. If you run just the first command, you should get only the PoS output either to a file (edit context.pbxt) or to stdout.

**Comment by ColinUNCG** [comment]
@smaransmarty  Thanks

**I only need output PoS tagging like this** 

> 1	Bob	_	NOUN	NNP	_	0		_	_
> 2	brought	_	VERB	VBD	_	0		_	_
> 3	the	_	DET	DT	_	0		_	_
> 4	pizza	_	NOUN	NN	_	0		_	_
> 5	to	_	ADP	IN	_	0		_	_
> 6	Alice	_	NOUN	NNP	_	0		_	_
> 7	.	_	.	.	_	0		_	_

**this is my demo.sh** 

> PARSER_EVAL=bazel-bin/syntaxnet/parser_eval
> MODEL_DIR=syntaxnet/models/parsey_mcparseface
> [[ "$1" == "--conll" ]] && INPUT_FORMAT=stdin-conll || INPUT_FORMAT=stdin
> 
> $PARSER_EVAL \
>   --input=$INPUT_FORMAT \
>   --output=stdout-conll \
>   --hidden_layer_sizes=64 \
>   --arg_prefix=brain_tagger \
>   --graph_builder=structured \
>   --task_context=$MODEL_DIR/context.pbtxt \
>   --model_path=$MODEL_DIR/tagger-params \
>   --slim_model \
>   --batch_size=1024 \

**however, the output has lots of  information like this**

> I syntaxnet/term_frequency_map.cc:101] Loaded 46 terms from syntaxnet/models/parsey_mcparseface/label-map.
> I syntaxnet/embedding_feature_extractor.cc:35] Features: input.digit input.hyphen; input.prefix(length="2") input(1).prefix(length="2") input(2).prefix(length="2") input(3).prefix(length="2") input(-1).prefix(length="2") input(-2).prefix(length="2") input(-3).prefix(length="2") input(-4).prefix(length="2"); input.prefix(length="3") input(1).prefix(length="3") input(2).prefix(length="3") input(3).prefix(length="3") input(-1).prefix(length="3") input(-2).prefix(length="3") input(-3).prefix(length="3") input(-4).prefix(length="3"); input.suffix(length="2") input(1).suffix(length="2") input(2).suffix(length="2") input(3).suffix(length="2") input(-1).suffix(length="2") input(-2).suffix(length="2") input(-3).suffix(length="2") input(-4).suffix(length="2"); input.suffix(length="3") input(1).suffix(length="3") input(2).suffix(length="3") input(3).suffix(length="3") input(-1).suffix(length="3") input(-2).suffix(length="3") input(-3).suffix(length="3") input(-4).suffix(length="3"); input.token.word input(1).token.word input(2).token.word input(3).token.word input(-1).token.word input(-2).token.word input(-3).token.word input(-4).token.word 
> I syntaxnet/embedding_feature_extractor.cc:36] Embedding names: other;prefix2;prefix3;suffix2;suffix3;words
> I syntaxnet/embedding_feature_extractor.cc:37] Embedding dims: 8;16;16;16;16;64
> I syntaxnet/term_frequency_map.cc:101] Loaded 64036 terms from syntaxnet/models/parsey_mcparseface/word-map.
> I syntaxnet/term_frequency_map.cc:101] Loaded 49 terms from syntaxnet/models/parsey_mcparseface/tag-map.
> INFO:tensorflow:Building training network with parameters: feature_sizes: [2 8 8 8 8 8] domain_sizes: [    5 10665 10665  8970  8970 64038]
> I syntaxnet/term_frequency_map.cc:101] Loaded 49 terms from syntaxnet/models/parsey_mcparseface/tag-map.
> I syntaxnet/term_frequency_map.cc:101] Loaded 46 terms from syntaxnet/models/parsey_mcparseface/label-map.
> I syntaxnet/embedding_feature_extractor.cc:35] Features: input.digit input.hyphen; input.prefix(length="2") input(1).prefix(length="2") input(2).prefix(length="2") input(3).prefix(length="2") input(-1).prefix(length="2") input(-2).prefix(length="2") input(-3).prefix(length="2") input(-4).prefix(length="2"); input.prefix(length="3") input(1).prefix(length="3") input(2).prefix(length="3") input(3).prefix(length="3") input(-1).prefix(length="3") input(-2).prefix(length="3") input(-3).prefix(length="3") input(-4).prefix(length="3"); input.suffix(length="2") input(1).suffix(length="2") input(2).suffix(length="2") input(3).suffix(length="2") input(-1).suffix(length="2") input(-2).suffix(length="2") input(-3).suffix(length="2") input(-4).suffix(length="2"); input.suffix(length="3") input(1).suffix(length="3") input(2).suffix(length="3") input(3).suffix(length="3") input(-1).suffix(length="3") input(-2).suffix(length="3") input(-3).suffix(length="3") input(-4).suffix(length="3"); input.token.word input(1).token.word input(2).token.word input(3).token.word input(-1).token.word input(-2).token.word input(-3).token.word input(-4).token.word 
> I syntaxnet/embedding_feature_extractor.cc:36] Embedding names: other;prefix2;prefix3;suffix2;suffix3;words
> I syntaxnet/embedding_feature_extractor.cc:37] Embedding dims: 8;16;16;16;16;64
> I syntaxnet/term_frequency_map.cc:101] Loaded 64036 terms from syntaxnet/models/parsey_mcparseface/word-map.
> INFO:tensorflow:Processed 1 documents
> 1	Bob	_	NOUN	NNP	_	0		_	_
> 2	brought	_	VERB	VBD	_	0		_	_
> 3	the	_	DET	DT	_	0		_	_
> 4	pizza	_	NOUN	NN	_	0		_	_
> 5	to	_	ADP	IN	_	0		_	_
> 6	Alice	_	NOUN	NNP	_	0		_	_
> 7	.	_	.	.	_	0		_	_
> 
> INFO:tensorflow:Total processed documents: 1
> INFO:tensorflow:num correct tokens: 0
> INFO:tensorflow:total tokens: 7
> INFO:tensorflow:Seconds elapsed in evaluation: 0.13, eval metric: 0.00%


**How can I  Only keep the PoS tagging info and remove the other info, 
my expected output like this** 

> 1	Bob	_	NOUN	NNP	_	0		_	_
> 2	brought	_	VERB	VBD	_	0		_	_
> 3	the	_	DET	DT	_	0		_	_
> 4	pizza	_	NOUN	NN	_	0		_	_
> 5	to	_	ADP	IN	_	0		_	_
> 6	Alice	_	NOUN	NNP	_	0		_	_
> 7	.	_	.	.	_	0		_	_

**Comment by prakhar21** [comment]
I have not tried it but was looking through the code and found that if you navigate to the directory **models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/syntaxnet/parser_eval.py**
Search for _logging.info_, may be commenting out these statements may work for you.

Just try it and let me know if it does not work or vice-versa. 

Thanks

**Comment by slavpetrov** [comment]
You probably want to use files as input/output instead of stdin/stdout. From the README:

Create or modify an input field inside the TaskSpec, with the file_pattern specifying the location we want. If the input corpus is in CONLL format, make sure to put record_format: 'conll-sentence'.
Change the --input and/or --output flag to use the name of the resource as the output, instead of stdin and stdout.
E.g., if we wanted to POS tag the CONLL corpus ./wsj.conll, we would create two entries, one for the input and one for the output:

input {
  name: 'wsj-data'
  record_format: 'conll-sentence'
  Part {
    file_pattern: './wsj.conll'
  }
}
input {
  name: 'wsj-data-tagged'
  record_format: 'conll-sentence'
  Part {
    file_pattern: './wsj-tagged.conll'
  }
}
Then we can use --input=wsj-data --output=wsj-data-tagged on the command line to specify reading and writing to these files.

You can uncomment the second call that performs parsing if you only care about POS tags. POS tagging should be a lot faster than POS tagging+parsing.
